### PR TITLE
Adding Pipeline and Trigger components

### PR DESCRIPTION
**Pull request by rhtevan**
Initial commit for the Pipeline and Trigger component, parameters and workspaces settings are to be adjusted according to the structure of shared file system.

Signed-off-by: Evan Zhang <evan.zhang@redhat.com>

### CODE REVIEW

File: pipelines/trigger-template-daas.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              claimName: daas-working-dir-pvc
+            subPath: source
+        pipelineSpec:
+          workspaces:
+            - name: source
+          tasks:
+            - name: fetch
+              params:
+                - name: url
+                  value: 'https://github.com/kiegroup/kogito-examples.git'
```
Comment (errantepiphany) [inline pipelines/trigger-template-daas.yaml:29]: This url is just for testing with, yes?

File: pipelines/trigger-template-daas.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                  value: 'dmn-quarkus-example'
+                - name: IMAGE_FULL_TAG
+                  value: 'image-registry.openshift-image-registry.svc:5000/kiegroup/daas-executor-ubi8-native:latest'
+              workspaces:
+                - name: source
+                  workspace: source
+              taskRef:
+                name: s2i-kogito
+                kind: ClusterTask
+
```
Comment (errantepiphany) [inline pipelines/trigger-template-daas.yaml:56]: Please remove 2 empty lines at bottom of file.

File: pipelines/trigger-template-daas.yaml (hunk 1 above)
Comment (rhtevan) [reply pipelines/trigger-template-daas.yaml:29]: The entire 'fetch' task will be removed once we connect it with 'working-dir'

File: pipelines/trigger-template-daas.yaml (hunk 2 above)
Comment (rhtevan) [reply pipelines/trigger-template-daas.yaml:56]: good point, will clean it.

Comment (rhtevan) [reply pipelines/trigger-template-daas.yaml]: done

**Pull request merged**